packages/server/src/routers/dev/index.ts

```diff
@@ -13,7 +14,7 @@ router.post(
   auth(),
   async (req: AuthenticatedRequest, res: Response, next: NextFunction) => {
     try {
-      if ((await PositionModel.find({})).length === 0)
+      if ((await PositionModel.find({})).length !== 0)
```

Comment:
There might be an easier way for devs to add data to the db manually. I'm not a fan of this route (even though I created it) Might remove in a separate PR😅

Reply:
a script might work well for this

Reply:
I agree. I'll remove in a follow up PR and come up with a useful script